warehouse/oidc/views.py

```diff
@@ -135,8 +139,16 @@ def mint_token_from_oidc(request: Request):
         )
 
     # Associate the given issuer claim with Warehouse's OIDCPublisherService.
+    # First, try the standard issuers
     service_name = OIDC_ISSUER_SERVICE_NAMES.get(unverified_issuer)
```

Comment:
If nobody ever uses a custom issuer, there's no lookup penalty, since it'll match the static dictionary value. This should be the case for most lookups.